components/automate-cli/cmd/chef-automate/automateHAAwsDeployment.go

```diff
@@ -205,7 +212,10 @@ func (a *awsDeployment) validateConfigFields() *list.List {
 func (a *awsDeployment) validateEnvFields() *list.List {
 	errorList := list.New()
 	if len(a.config.Aws.Config.Profile) < 1 {
-		errorList.PushBack("Invalid or empty aws profile name")
+		check, _ := a.getBastionIamRole()
```

Comment:
`getBastionIamRole()` is a function returning bool value. Can we use a better name instead of check. Maybe `isIamRolePresent` ?

Reply:
Renamed to the function as suggested `isIamRolePresent`